### DISCUSSION TITLE
NoiseModelling results do not match the observation at midnight

### DISCUSSION THREAD

**Discussion by zjc9988** [Universite-Gustave-Eiffel/NoiseModelling#545] [Q&A]
Dear NoiseModelling group:

  I am using NoiseModelling-4.0.0 for a traffic noise simulation. All the results come out well except for the results at midnight, namely at about 2-4 a.m. The observations are between 50-55dB (Leq), but I never get a result lower than 55dB for the receiver at the monitor point, no matter how I change the traffic. I wonder if there are any other factors controlling the background noise level in NoiseModelling?

  Thanks for your reply.

**Comment by pierromond** [comment]
If it is a the monitoring point, it is really strange, it can be a bug. Can you provide more information ? Maybe a picture of your map ?

**Comment by zjc9988** [comment]
[input.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/10823571/input.zip)
Part of the input files are attached here.
The receiver at the monitoring point is ID 57582.
Thanks for the help. @pierromond

**Comment by pierromond** [comment]
MMM it seems your input are ok. Do you have the output file ?

**Reply by zjc9988** [reply]
[LNIGHT_GEOM.zip](https://github.com/Universite-Gustave-Eiffel/NoiseModelling/files/10824307/LNIGHT_GEOM.zip)
The LNIGHT files are attached here.

**Comment by pierromond** [comment]
mmm By my side, I don't see any problem...

**Reply by zjc9988** [reply]
Well, I think the model is alright because I get quite good match for day and evening hours. However, even if I set the traffic at night to 1 and the speed at very low value, I still get about 56dB at my monitoring point. I just wonder if changing any other input values could affect the sound level? I'm not sure if it comes from some kind of background noise settings?

**Reply by pierromond** [reply]
The minimum speed in CNOSSOS-EU is 20km/h. So if you have one car by hour, with a speed of 20 km/h, I can imagine that CNOSSOS give a LWDEN value close to 60 dBA. There is no background noise in the computation. You can look about the LW segments using the Road_Emission_From_Traffic bloc if you want.
